worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/IChunkSet.java

```diff
@@ -30,7 +30,7 @@ default boolean setBiome(BlockVector3 position, BiomeType biome) {
     @Override
     <T extends BlockStateHolder<T>> boolean setBlock(int x, int y, int z, T holder);
 
-    void setBlocks(int layer, char[] data);
+    void setBlocks(int layer, Object data);
```

Comment:
I'm not a fan of this. Given the methods are split between char and int in numerous other places, it might be worth doing so here as well.

Reply:
I have split it into a ```setIntBlocks()``` and ```setCharBlocks()``` set of methods and adjusted to code in other places as needed.